### PR TITLE
add json file for Bruno

### PR DESCRIPTION
**Pull request by insectengine**
** If you are updating a guide, please submit your pull request to the main repository: https://github.com/quarkusio/quarkus/tree/main/docs/src/main/asciidoc **

### CODE REVIEW

**Review by brunobat — APPROVED** [review]

**Comment by gsmet** [comment]
@brunobat could you clarify what it is used for?

File: assets/json/03656937-19FD-4C83-9066-C76631D445EA.json [hunk 1]
```diff
@@ -0,0 +1 @@
+{"active":true,"deny_anonymous_ids":[],"deny_quarkus_versions":[],"refresh_interval":43200.000000000}
```
Comment (maxandersen) [inline assets/json/03656937-19FD-4C83-9066-C76631D445EA.json:1]: should this not be enabled true?

**Review by gsmet — APPROVED** [review]
I will merge it to unblock things but I would like to understand why we need this file.

**Comment by maxandersen** [comment]
> @brunobat could you clarify what it is used for?

its the file build analytics fetches to know if analytics is still active for the specific quarkus version.

i.e. imagine some bug causes data issues we can disable that specific version or some day we have to change the collection - then instead of old quarkus builds keeps pushing anonymous data they will simply stop.

This file was before on a gist - I suggested to move it under quarkus.io so its all as transparent as possible.

File: assets/json/03656937-19FD-4C83-9066-C76631D445EA.json (hunk 1 above)
Comment (maxandersen) [reply assets/json/03656937-19FD-4C83-9066-C76631D445EA.json:1]: also, wouldn't it make more sense to have this under `quarkus.io/uages/xx-yyy.json` ?

Comment (brunobat) [reply assets/json/03656937-19FD-4C83-9066-C76631D445EA.json:1]: this is remote config for us... We don't instruct users to change that.

**Comment by gsmet** [comment]
@brunobat and it won't break us in case of a GitHub outage, right?

**Comment by brunobat** [comment]
The builds themselves will check each 12h, if the file is not found will inactivate analytics and will try later.

**Comment by maxandersen** [comment]
should fail? you mean should NOT fail, right?

**Comment by brunobat** [comment]
It will actually fail because the remote config file is kind of important. 
I can do make it a manual test...

**Comment by maxandersen** [comment]
We should fix that asap. Just because quarkus can't get info about its analytics should not prevent users from building their project.

**Comment by brunobat** [comment]
User's project will continue to build.
It's the Quarkus project itself that has a test that will fail if the config is not there.

**Pull request merged**